webcompat/static/js/lib/bugform.js

```diff
@@ -159,10 +180,11 @@ function BugForm() {
       $("[value=" + problemType[1] + "]").click();
     }
 
-    // If we got a details param, add that to the end of the issue description.
+    // If we got a details param, add that to the end of the steps to reproduce field
+    // github.com/webcompat/webcompat.com/issues/1547#issuecomment-305649762
```

Comment:
@miketaylr replying here to your comment below:
>I kind of assume when a developer does something weird or breaks the pattern, it is a workaround for some unwanted behavior. but I am only guessing at best what was the original developer's intent.

>I'm not sure what this refers to (or if I need to at this point :p). What was the weird pattern?

I think you already responded to the conversation. we were debating why that outer label for problem_type seemed to be defined in form.py (and not used) and then hard-coded in form.html. and whether that was significant to preserve.